gap/oper.gi

```diff
@@ -1684,3 +1684,43 @@ function(digraph, list)
 
   return DigraphShortestDistance(digraph, list[1], list[2]);
 end);
+
+#
+
+InstallMethod(DigraphClosure,
+"for a digraph and an integer",
+[IsDigraph, IsPosInt],
+function(digraph, k)
+  local out, adj_mat, degs, n, new_edge, row, i, j;
+
+  if not IsSymmetricDigraph(digraph) or DigraphHasLoops(digraph)
+      or IsMultiDigraph(digraph) then
+    ErrorNoReturn("Digraphs: DigraphClosure: usage,\n",
+      "the digraph must by symmetric, without loops, and no multiple edges,");
```

Comment:
The linter isn't yet set up to catch these kinds of problems. This line should be aligned with the one above:

```
    ErrorNoReturn("Digraphs: DigraphClosure: usage,\n",
                           "the digraph...
```
However, this makes your line too long, so you'll need to break it, so it's something like"
```
    ErrorNoReturn("Digraphs: DigraphClosure: usage,\n",
                  "the digraph must by symmetric, without loops, and no ",
                  "multiple edges,");
```